### PR TITLE
Small style fixes

### PR DESCRIPTION
**Pull request by danimoh**

### CODE REVIEW

File: src/request/create/CreateLow.js [hunk 1]
```diff
@@ -46,17 +46,17 @@ class CreateLow {
         this._pinInput.on(
             PinInput.Events.PIN_ENTERED,
             /** @param {number} pin */ async pin => {
-                $confirmMessage.classList.add('hidden');
-                $notMatchingMessage.classList.add('hidden');
+                $confirmMessage.classList.add('display-none');
```
Comment (sisou) [inline src/request/create/CreateLow.js]: I believe the `hidden` class is used here on purpose to not make the content jump when unhiding it.

File: src/request/create/index.html [hunk 2]
```diff
@@ -42,7 +42,7 @@
     <link rel="stylesheet" href="../../components/PassphraseInput.css">
     <link rel="stylesheet" href="../../components/RecoveryWords.css">
     <link rel="stylesheet" href="../../components/ValidateWords.css">
-    <link rel="stylesheet" href="./privacy-agent.css">
+    <link rel="stylesheet" href="../../components/PrivacyAgent.css">
```
Comment (sisou) [inline src/request/create/index.html]: This is already fixed in Sven's PR.

Comment (sisou) [reply src/request/create/index.html]: Or rather the CSS is already added some lines above.

File: src/request/create/privacy-agent.css [hunk 3]
```diff
@@ -1,52 +0,0 @@
-.privacy-agent {
```
Comment (sisou) [inline src/request/create/privacy-agent.css]: @svub did you not remove this file?

File: src/request/create/CreateLow.js (hunk 1 above)
Comment (danimoh) [reply src/request/create/CreateLow.js]: To keep the no-jumping behaviour the message is absolutely positioned in an always visible container.

I changed the visibility:hidden to display:none two avoid that:
- both messages take space in the layout while at most one is visible at the same time
- the messages are not rendered at the same position
- the bottom message has no margin to the pinpad dots

File: src/request/create/index.html (hunk 2 above)
Comment (danimoh) [reply src/request/create/index.html]: You mean #54 ?
Apparently removal of this file was forgotten there

**Pull request merged**